modules/cluster-limitations-local-zone.adoc

```diff
@@ -17,3 +17,14 @@ The following list details limitations when deploying a cluster in AWS Local Zon
 - Network resources such as Network Load Balancer (NLB), Classic Load Balancer, and Network Address Translation (NAT) Gateways are not supported in AWS Local Zones.
 - For an {product-title} cluster on AWS, the AWS Elastic Block Storage (EBS) `gp3` type volume is the default for node volumes and the default for the storage class. This volume type is not globally available on Local Zone locations. By default, the nodes running in Local Zones are deployed with the `gp2` EBS volume. The `gp2-csi` `StorageClass` must be set when creating workloads on Local Zone nodes.
 ====
+
+If you want the installation program to automatically create Local Zone subnets for your {product-title} cluster, specific configuration limitations apply with this method.
+
+[IMPORTANT]
+====
+The following list details configuration limitations that apply when you set the installation program to automatically create subnets for your {product-title} cluster:
+
+- A network plugin, which is specific to an overlay network, automatically adjusts the MTU value for an overlay network. With the installation program automatically creating subnets on your behalf, you cannot adjust the MTU value for the overlay network.
+- A network plugin uses the Network Address Translation (NAT) Gateway from the route table of the parent zone to route egress traffic for private subnets in an AWS Local Zone. If a parent zone does not exist in your VPC, the network plugin uses the NAT gateway listed in the first available private subnet listed in an AWS region.
```

Comment:
@dfitzmau could you please elaborate that part? The `network plugin` is completely different than NAT Gateway limitation. I can't connect the items in this paragraph.

I would suggest rewriting the entire paragraph to something like this:

- The private subnets in Local Zone are associated with the route table of the parent zone to route egress traffic to the internet. If that route table is not present when installing a cluster, the subnet will be associated with the first private route table available in the VPC. This approach is valid only for Local Zones subnets in OpenShift cluster, used to keep the nodes in private subnets while AWS does not support NAT gateways globally in those locations.

Reply:
Yes, these changes were introduced through https://github.com/openshift/openshift-docs/pull/57736 . I can update this section with your suggested text.

Reply:
1. The private subnets in an AWS Local Zone > The private subnets for an AWS Local Zone ?

2. If the route table > If this route table ("this" forming a stronger bond than "that" for the previous sentence).

3. I prefer to avoid future tense in the event something does not hold through. "will be associated" can mean the same thing as "associates". A case of future tense versus present tense. I'm also trying to remove as much passive voice (is/are/be) as this can hide the agent doing the work by using more active voice.

Reply:
Original statement:

> This approach is valid only for Local Zones subnets in OpenShift cluster, used to **keep the nodes in private subnets** while AWS does not support NAT gateways globally in those locations.

Reply:
> I also didn't follow that part. I am trying to say the service availability. The NAT GW is not available across all Local zones, only in some. https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/

> because AWS does not support Network Address Translation (NAT) Gateways across all AWS Local Zone locations.

?

Reply:
I think this comment was resolved with the next reviews

Reply:
Yes, correct. I'll mark this thread as resolved.